lib/matplotlib/dates.pyi

```diff
@@ -0,0 +1,200 @@
+import datetime
+import numpy as np
```

Comment:
NumPy is third-party and thus should be in the second block between stdlib and first-party imports.